PlayCover/Views/Settings/IPASourceSettings.swift

```diff
@@ -102,11 +103,16 @@ struct IPASourceSettings: View {
 
 struct SourceView: View {
     var source: SourceData
+    @State var isEnabled: Bool
     @State var showingPopover = false
 
     var body: some View {
         HStack {
-            Text(source.source)
+            Toggle("", isOn: $isEnabled).onChange(of: isEnabled) { value in
+                StoreVM.shared.enableSourceToggle(source: source, value: value)
+            }
+            .help("state.enabled")
```

Comment:
Is `state.enabled` already in the localisation file?

Reply:
Yes: https://github.com/PlayCover/PlayCover/blob/9a66b3a5ec0bec4f181529353045022acc9d6c57/PlayCover/ca.lproj/Localizable.strings#L243